VictoryCenter/VictoryCenter.WebAPI/Controllers/BaseApiController.cs

```diff
@@ -15,17 +16,44 @@ public class BaseApiController : ControllerBase
     
     protected ActionResult HandleResult<T>(Result<T> result)
     {
+        var problemsFactory = HttpContext.RequestServices
+            .GetRequiredService<ProblemDetailsFactory>();
+
         if (result.IsSuccess)
         {
-            return (result.Value is null) ?
-                NotFound("Not Found") : Ok(result.Value);
+            if (result.Value is not null)
+            {
+                return Ok(result.Value);
+            }
+
+            var notFoundDetails = problemsFactory.CreateProblemDetails(
+                HttpContext,
+                statusCode: StatusCodes.Status404NotFound,
+                title: "Not Found",
```

Comment:
Can we move these general exception titles and messages to separate place (like static class or something better)?